packages/web-api/src/auth/routes.ts

```diff
@@ -32,20 +37,41 @@ import {
   getRefreshTokenObject,
   isRefreshTokenValid as validateRefreshToken
 } from './utils';
-import z from 'zod';
-import { PreApprovedUserService } from '../services/preApproval';
 
 require('express-async-errors');
 export const router: Router = express.Router();
 
 // All users are granted this role by default
 export const BASE_ROLES: UserRole[] = [UserRole.DEFAULT];
 
+// Rate limiting for password routes
+const authRateLimit = (per15Minutes: number) => {
```

Comment:
why not just have `max` and `windowMs` as params? the hardcoding of 15 minutes seems arbitrary